corehq/apps/export/models/new.py

```diff
@@ -792,6 +800,76 @@ def _new_from_schema(cls, schema):
         )
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = SMS_EXPORT
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        return cls(
+            domain=schema.domain,
+            tables=[
+                TableConfiguration(
+                    label="Messages",
+                    columns=[
+                        RowNumberColumn(),
+                        ExportColumn(
+                            label="Contact ID",
```

Comment:
the current export also has a column for Contact Type, which takes the value of one of:

Mobile Worker
Case
Web User
Unknown